build-tools/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -220,5 +220,11 @@
         <module name="CyclomaticComplexity"/>
         <module name="JavaNCSS" />
         <module name="NPathComplexity"/>
+
+        <module name="FinalParametersCheck">
+            <property name="severity"
+                      value="${checkstyle.finalparameter.severity}"
```

Comment:
nit: `checkstyle.finalparameters.severity`, (not the 's' on the end of parameters)